core/src/main/java/org/renjin/primitives/special/SubstituteFunction.java

```diff
@@ -56,7 +56,7 @@ public SEXP apply(Context context, Environment rho, FunctionCall call, PairList
     SEXP expr;
     if(exprArgument == Symbols.ELLIPSES) {
       
-      SEXP ellipses = rho.getVariable(Symbols.ELLIPSES);
+      SEXP ellipses = rho.getVariable(context, Symbols.ELLIPSES);

```

Comment:
I don't think GNU R allows you to define an active binding for the symbol `...`. At least the following fails:

```
makeActiveBinding(sym=`...`, fun=function(...) 42)
```

So we can add a `getEllipsesVariable()` method to the environment which in turn calls `getVariableUnsafe()`.